pkg/controller/storageversionmigrator/storageversionmigrator.go

```diff
@@ -271,6 +270,15 @@ func (svmc *SVMController) sync(ctx context.Context, key string) error {
 		if err != nil {
 			return err
 		}
+		rvCmp, err := resourceversion.CompareResourceVersion(accessor.GetResourceVersion(), listResourceVersion)
+		if err != nil {
+			logger.V(4).Error(err, "Unable to compare the resource version of the resource", "namespace", accessor.GetNamespace(), "name", accessor.GetName(), "gvr", gvr.String(), "error", err.Error())
+			return err
+		}
+		if rvCmp == 1 {
+			logger.V(6).Info("Resource ignored due to resource version being greater than the SVM checkpoint", "namespace", accessor.GetNamespace(), "name", accessor.GetName(), "gvr", gvr.String())
```

Comment:
May be helpful to include `accessor.GetResourceVersion()` and `listResourceVersion` in the log messages.

Reply:
Good point, added!